indico/migrations/versions/20190823_1438_0f20a5e4018e_add_new_abstract_state.py

```diff
@@ -0,0 +1,34 @@
+"""Add new abstract state
+
+Revision ID: 0f20a5e4018e
+Revises: 620b312814f3
+Create Date: 2019-08-23 14:38:34.322010
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '0f20a5e4018e'
+down_revision = '620b312814f3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute('''
+    ALTER TABLE event_abstracts.abstracts DROP CONSTRAINT ck_abstracts_valid_enum_state;
```

Comment:
I think in all other alembic revisions we indented the contents of those strings.